bsondump/bsondump.go

```diff
@@ -66,9 +67,9 @@ func (bdo *BSONDumpOptions) GetWriter() (io.WriteCloser, error) {
 	return WriteNopCloser{os.Stdout}, nil
 }
 
-// GetBSONReader opens and returns an io.ReadCloser for the BSONFileName in BSONDumpOptions
+// GetBSONReader opens and returns an io.ReadCloser for the BSONFileName in OutputOptions
 // or nil if none is set. The caller is responsible for closing it.
-func (bdo *BSONDumpOptions) GetBSONReader() (io.ReadCloser, error) {
+func (bdo *OutputOptions) GetBSONReader() (io.ReadCloser, error) {
```

Comment:
nit: From a readability perspective, I think "bdo" should be renamed to match the new struct name.